bigframes/display/table_widget.js

```diff
@@ -1,5 +1,5 @@
-/**
- * Copyright 2025 Google LLC
+/*
+ * Copyright 2024 Google LLC
```

Comment:
We shouldn't update the copyright year.

```suggestion
 * Copyright 2025 Google LLC
```